dbcon/joblist/batchprimitiveprocessor-jl.cpp

```diff
@@ -890,7 +890,8 @@ void BatchPrimitiveProcessorJL::getRowGroupData(ByteStream& in, vector<RGData>*
 
         if (joinResults.get() == NULL)
         {
-          joinResults.reset(new vector<uint32_t>[8192]);
+          auto v = new vector<uint32_t>[8192];
+          joinResults.reset(v);
```

Comment:
I don't see any issue with the original code. Is this change related to the `set::contains()` change?

Reply:
This is changes serg made for getting rid of buggy compiler warnings on Ubuntu 22.10. I'm ok with it.

Reply:
Thanks for clarifying.